addons/web/static/src/webclient/icons.scss

```
@@ -75,3 +74,16 @@ $oi-sizes: (
     mask: url('/web/static/img/barcode.svg') center/contain no-repeat;
     background-color: $o-brand-primary;
 }
+
+// This is rtl language specific fix. It will flip the icons 180 degree
+// ----------------------------------------------------------------------------
+.o_rtl .oi.oi-chevron-left,
```

Is it really necessary to add the `.oi` main class ? eg:
```suggestion
.o_rtl .oi-chevron-left,
```